src/dialogs/examples/basic/O365ConnectorCardActionsDialogV2.ts

```diff
@@ -0,0 +1,119 @@
+import * as builder from "botbuilder";
+import { TriggerActionDialog } from "../../../utils/TriggerActionDialog";
+import { DialogIds } from "../../../utils/DialogIds";
+import { DialogMatches } from "../../../utils/DialogMatches";
+import { Strings } from "../../../locale/locale";
+import * as teams from "botbuilder-teams";
+
+export class O365ConnectorCardActionsDialogV2 extends TriggerActionDialog {
```

Comment:
At first glance these examples look the same. Can you add a top-level comment to the file describing what you are trying to show in each dialog?

Reply:
Done.